README.md

```diff
@@ -1 +1,86 @@
-# Capstone-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CodeFire WebPage 
+
+> This project is a website consiting of only two pages, home and about, it works with CSS, HTML and JS. Project for Microverse Capstone 1
+
+![screenshot](./screenshot.png)
+
+Currently it is only a really simple website for the Microverse Capstone 1, however, it will be upgraded in the future with more content.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+
+## Live Demo
```

Comment:
**Kindly provide a link to your Video presentation**
- Provide feedback about the video presentation in the code review summary.

1. Record a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video needs to have a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
4. The video needs to have a demo of the project, showing the pages in different screen sizes.
5. The video needs to have a highlight about a piece of the code the student finds interesting or is proud of making.